### PR TITLE
Adds a lot of new drinks and bottles and the TCD

### PR DESCRIPTION
**Pull request by dovydas12345**
## What Does This PR Do

- Adds about 90 new drinks and bottles combined.
- Adds sprites to coffee and tea, so they are not just normal brown glass.
- Adds a Temperature Control Device, which depending on its setting either cools or heats up the reagents inside a container. When heating up reagents inside the container it heats them up to 500 K (226.85 C) and when cooling reagents it only lowers the temperature down to 250 K (-23.15 C)
- Fixes spelling in descriptions of wine, ale and cognac.

Credits:
Thanks to @LilyCandyKitten for the sprites.
Thanks to @PidgeyThePirate for adding most of the drinks and the TCD.
Thanks to @Ionward for the TCD sprites. 

## Why It's Good For The Game

Adds a lot variety to the game making it more unique. More fun for the bartender and bar visitors. 

## Images of changes

Sprites of new bottles and their drinks in a glass:

Sprites of new juice boxes:

Sprites of new cocktails:

TCD sprites:

tcd1

tcd2

## Changelog
:cl:
add: Added about 90 new drinks and bottles combined.
add: Added the temperature control device, which heats up or cools reagents inside a container.
add: Sprites for coffee and tea in a glass.
spellcheck: Fixed the spelling of alcoholic in the descriptions of wine, ale and cognac.
/:cl:

### CODE REVIEW

**Comment by Carthusia** [comment]
Very nice, are these brand new drinks and bottles, or are they replacers for the current ones?

**Comment by dovydas12345** [comment]
All of them are brand new except for 2 sprites that are for coffee and tea.

**Comment by Carthusia** [comment]
Nice, I think the wiki is going to require an update for all these to let people know how to mix them and such.
https://i.gyazo.com/f1aa511fcce9aa17d92518454555e170.png
This one looks like chocolate syrup and i love that fact

**Comment by taukausanake** [comment]
> Temperature Control Device

So this is just a handheld and cooling version of the [chemical heater](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/reagents/chemistry/machinery/chem_heater.dm)? Not sure which I would like more but I'm willing to give this a look

File: code/game/objects/items/devices/temperature_control_device.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	icon_state = "tcd-hot"
+	item_state = "tcd-hot"
+	w_class = WEIGHT_CLASS_SMALL
+	slot_flags = SLOT_BELT
+	materials = list(MAT_METAL=200, MAT_GLASS=200)
+	force = 2
+	throwforce = 0
+	var/actingtemp = 500
+	var/hot_temp = 500
+	var/cold_temp = 250
```
Comment (moxian) [inline code/game/objects/items/devices/temperature_control_device.dm]: The numbers here don't match those in the PR description (neither heating nor cooling, but i guess the former weren't meant to be exact anyway so it's fine).
Please realign (at least the cooling ones).

Comment (dovydas12345) [reply code/game/objects/items/devices/temperature_control_device.dm]: Updated the PR description to be the same as the values assigned.

**Comment by Fox-McCloud** [comment]
These sprites don't look very good at all; they're quite low quality compared to most of the drink sprites we have---and they certainly don't fit well either.

The device I'm not really sold on; if we want the bartender to be able to heal/cool things, then just give him a chem heater; I really don't like the idea of a device that cools, on demand.

**Comment by Pewtershmitz** [comment]
I'd like to help with improving the sprites. Most of the -really- need outlines.
But generally, I like them. As for the device to cool stuff, I don't really see a point to it. I know some recipes require heating, but I'd rather just have a hot plate on the counter or something like that!

Please contact me on discord @Pewtershmitz If you're interested in improving the sprites!

**Comment by dovydas12345** [comment]
Doesn't seem like the sprites are getting improved. Don't see it going anywhere.

**Pull request closed without merging**